src/blueprints/organization/templates/organization.html

```diff
@@ -1,14 +1,29 @@
 {% extends 'base.html' %}
 
 {% block header %}
-  <h1>{% block title %}Hello World{% endblock %}</h1>
+  <h1>{% block title %}[Organization] View{% endblock %}</h1>
   <link rel="stylesheet" href="{{ url_for('bp_organization.static', filename='css/styles.css') }}">
```

Comment:
I'm going to update the base template so that we can insert this stuff directly into the head tag.
I will do that when I refactor the templates that we have created